modules/images-describing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-describing-image-registry-settings-hcp_{context}"]
+= Describing image registry settings for ROSA with HCP
+
+
+Description needed !!!!
```

Comment:
Verify that your image registry is enabled in the cluster details by running the following command:

Reply:
integrated the describe module into the create module doc